### ISSUE TITLE
Allow to set mode for downloadable translations

### ISSUE THREAD

**Issue by rmehner** [eHealthAfrica/grunt-tx#3]
Transifex allows to have translations reviewed and only have those downloaded. `grunt-tx` should support that. See http://docs.transifex.com/api/translations#get for details

**Comment by seb86** [comment]
Any update on this issue?

**Comment by rmehner** [comment]
@seb86 nope, not actively using this project anymore and don't have the resources to maintain this. Sorry.

**Comment by seb86** [comment]
Is there a better one that you use or know of?

**Comment by rmehner** [comment]
IIRC @tlvince did something similar the other day by only using cURL and not having most of the whole boilerplate that this project has. But I'm not informed on the latest developments on this.

Last time I've used transifex was in a different context and I've build something myself, but unfortunately that's closed source.

**Comment by seb86** [comment]
OK
